bin/decode_dist.R

```diff
@@ -83,34 +83,6 @@ AdjustCounts <- function(counts, params) {
   })
 }
 
-ValidateInput <- function(params, counts, map) {
```

Comment:
It is moved to Decode(), as CheckDecodeInputs.  The change description
mentions that.

Andy


On Fri, Jan 8, 2016 at 1:47 PM, Tom Kaitchuck <notifications@github.com>
wrote:

> In bin/decode_dist.R
> <https://github.com/google/rappor/pull/60#discussion_r49242256>:
>
> > @@ -83,34 +83,6 @@ AdjustCounts <- function(counts, params) {
> >    })
> >  }
> >
> > -ValidateInput <- function(params, counts, map) {
>
> Why is this being removed?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/google/rappor/pull/60/files#r49242256>.
>